code/game/objects/effects/decals/crayon.dm

```diff
@@ -666,29 +735,129 @@ obj/item/scroll/attackby(obj/item/I, mob/living/carbon/human/M)
 		qdel(frodo)
 	return
 
-//ritual knife spell procs
+// Scribe: Gives you the Scribe perk. This is a requirement for inscribing scrolls with the spells below.
+/obj/effect/decal/cleanable/crayon/proc/scribe_spell(mob/living/carbon/human/M)
+	var/datum/reagent/organic/blood/B = M.get_blood()
+	var/mob/living/carbon/human/H = M
+	for(var/obj/item/paper/P in oview(1)) // Requires a paper
+		if(H.stats.getPerk(PERK_ALCHEMY))
+			to_chat(M, SPAN_WARNING("The paths of the Scribe and the Alchemist are mutually exclusive."))
+			return
+		M.maxHealth -= 25
+		M.health -= 25
+		B.remove_self(100)
+		M.stats.addPerk(PERK_SCRIBE)
+		M.sanity.changeLevel(20)
+		to_chat(M, "<span class='warning'>Your head throbs like a heartbeat, the sudden insight of knowledge on how to pen down your dissasociated thoughts into scrolls fogs your eyes, until you can see no more.</span>")
+		qdel(P)
+	return
+
+// Awaken: Evolve our Ritual Knife into a greater form.
+// It can be used once more upon the harvester for it to become a blade
+/obj/effect/decal/cleanable/crayon/proc/awaken_spell(mob/living/carbon/human/M)
+	var/datum/reagent/organic/blood/B = M.get_blood()
+	for(var/obj/item/tool/knife/ritual/knifey in oview(1)) // No ascending church knives
+		if(istype(knifey, /obj/item/tool/knife/ritual) && !istype(knifey, /obj/item/tool/knife/ritual/sickle) && !istype(knifey, /obj/item/tool/knife/ritual/blade))
+			to_chat(M, "<span class='warning'>Your weapon twists its form, metal bending as if it were flesh with a sickening crunch!</span>")
+			playsound(loc, 'sound/items/biotransform.ogg', 50)
+			new /obj/item/tool/knife/ritual/sickle(knifey.loc)
+			B.remove_self(100)
+			M.maxHealth -= 5
+			M.health -= 5
+			M.sanity.changeLevel(5)
```

Comment:
why do these restore sanity?

Reply:
I'm not versed on sanity, I had to copy from the original file on upstream to imitate, and I saw the Scribe spell changing this value by 20 positive, you mean to tell me positive values are GOOD sanity?

Reply:
Mhm